web/docs/operators/schemas.md

```diff
@@ -0,0 +1,28 @@
+---
+sidebar_custom_props:
+  operator:
+    source: true
+---
+
+# schemas
+
+Retrieves all schemas for events stored at a node.
+
+## Synopsis
+
+```
+schemas
+```
+
+## Description
+
+The `schemas` operator shows all schemas of all events stored at a node.
+
+## Examples
+
+See all available schema definitions for the schema `suricata.alert`
```

Comment:
Should we explain to users that its possible for events to e.g. have more fields than the original schema specified?

Otherwise this may read a bit strange, as "surely there can only be one `suricata.alert`"

Reply:
I added a note under _Description_ for this.

Reply:
Done in 3f0546a2b76.